torch/distributed/fsdp/flat_param.py

```diff
@@ -87,6 +87,9 @@
 # pre-backward each iteration.
 _FSDP_SKIP_WRITEBACK_CHECK = "FSDP_SKIP_WRITEBACK_CHECK"
 
+# Env var toggling whether when model is in .eval() mode, should we run in fp32
+# or the reduced precision.
+_FSDP_FULL_PREC_IN_EVAL = "FSDP_FULL_PREC_IN_EVAL"
```

Comment:
nit: Should the naming be `_FSDP_EVAL_IN_FULL_PREC = "FSDP_FULL_PREC_IN_EVAL"` or `_FSDP_USE_FULL_PREC_IN_EVAL = "FSDP_USE_FULL_PREC_IN_EVAL"`?

My reasoning here is that you save this to an attribute `_full_prec_in_eval: bool`, so I would read it like "if full precision in eval". I read it more naturally as "if eval in full precision" or "if use full precision in eval".

Reply:
sounds good!